inc/features/blocks/namespace.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Experience Block functions.
+ *
+ * @package altis-experiments
+ */
+
+namespace Altis\Experiments\Features\Blocks;
+
+/**
+ * Include and set up Experience Blocks.
+ */
+function setup() {
+	require_once __DIR__ . '/personalization/register.php';
+	require_once __DIR__ . '/personalization-variant/register.php';
+
+	// Register blocks.
+	Personalization\setup();
+	Personalization_Variant\setup();
+
+	// Register experience block category.
+	add_filter( 'block_categories', __NAMESPACE__ . '\\add_block_category', 9 );
+}
+
+/**
+ * Adds an experience block category to the block editor.
+ *
+ * @param array $categories Array of block editor block type categories.
+ * @return array The modified block categories array.
+ */
+function add_block_category( array $categories ) : array {
+	$categories[] = [
+		'slug' => 'altis-experience-blocks',
+		'title' => __( 'Experience Blocks', 'altis-experiments' ),
+	];
+
+	return $categories;
+}
+
+/**
+ * Reads and returns a block.json file to pass shared settings
+ * between JS and PHP to the register blocks functions.
+ *
+ * @param string $name The directory name of the block relative to this file.
+ * @return array|null The JSON data as an associative array or null on error.
+ */
+function get_block_settings( string $name ) : ?array {
```

Comment:
Is there a core function for this? 🤔

Reply:
Probably in 5.4 but not that I could find. I'm not sure how long core has been following the `block.json` pattern.